src/chat-panel/components/ChatPanel.tsx

```diff
@@ -82,12 +83,19 @@ class _ChatPanel extends Component<Props, State> {
       <div
         className={this.props.activeUser!.id === message.userId ? 'messageDivActive' : 'messageDiv'}
       >
+        <div className="messageEdit" onClick={this.onMessageDeletion(message.id)}>
```

Comment:
This should really be a `<button />` rather than a `<div />`, for accessibility reasons.
You'll need to remove the default button styling to do so, but it's a lot more syntactically correct.